src/test/java/study/race/domain/TestCarName.java

```diff
@@ -0,0 +1,44 @@
+package study.race.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import study.race.domain.CarName;
+
+public class TestCarName {
+
+
+    @ParameterizedTest(name = "CarName 객체 - null 값 테스트")
+    @NullAndEmptySource
+    public void test_carname_constructor_null(String testCarName) {
+        assertThatThrownBy(() -> new CarName(null))
```

Comment:
다음과 같이 testCarName를 사용해야겠네요 😄 
```suggestion
        assertThatThrownBy(() -> new CarName(testCarName))
```